### ISSUE TITLE
ANN202 reported on the wrong column

### ISSUE THREAD

**Issue by sumnerevans** [python-discord/flake8-annotations#70]
**Describe the bug**
The `ANN202` is reported in the incorrect column location when the last parameter has a type annotation.

**To Reproduce**
Minimal code example to reproduce the behavior:

```py
def foo(bar: int):
    return 1


def bar():
    return 1
```

Results:
```
test.py:1:13: ANN201 Missing return type annotation for public function
test.py:5:11: ANN201 Missing return type annotation for public function
```

Expected: the first one should have a column number of `17` or `18`.

**Version Information**
Please provide the full output of `flake8 --version`

```bash
$ flake8 --version
3.7.9 (flake8-annotations: 2.0.0, flake8-comprehensions: 3.2.2, flake8-print: 3.1.4, flake8_pep3101: 1.2.1, import-order: 0.18.1, mccabe: 0.6.1, pycodestyle: 2.5.0, pyflakes: 2.1.1) CPython 3.8.1 on Linux
```

As well as your Python version:
```bash
$ python -V
Python 3.8.1
```

**Comment by sumnerevans** [comment]
If it helps, looking at this more, it appears like the error is always marked as being right after the first parameter. Example:
```python
        def row_score(key: str, row_items: Iterable[str]):
            return max(map(lambda m: m[1], process.extract(key, row_items)))
```
the error shows up between the `:` and `s`.

**Comment by sumnerevans** [comment]
Thanks for fixing this so quickly! Confirming that this works for me.
